credhub

```diff
@@ -0,0 +1,9 @@
+---
+syntax: bash
+tags: [ credhub ]
+---
+# To set a CredHub credential, of type 'value', using the contents of a file:
+credhub set -n <credential-path> -t value -v "$(cat <file-path>)"
```

Comment:
```suggestion
# To set a CredHub credential, of type `value`, using the contents of a specific file:
credhub set -name <credential-path> -type <password|user|certificate|ssh|rsa> -value "$(cat <in-file>)"
```

Optional fix: `specific` may sound better.
Optional fix: long options can be used for readability.
Optional fix: `-type` values can be inlined.
Optional fix: path placeholders can be standardized according to [this](https://github.com/cheat/cheatsheets/pull/194) PR.